src/components/Card/Card.module.scss

```diff
@@ -0,0 +1,18 @@
+@import 'src/styles/settings.scss';
+
+.c-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  padding: $space-xl;
+  border: 1px solid $color-medium-purple;
+  border-radius: $space-s;
+  margin: $space-m;
+
+  box-shadow: $color-light-purple 0px 5px 15px;
```

Comment:
Create variable for shadow and re-use it or use variables instead of pixel values